AmplifyTools/AmplifyXcode/Sources/AmplifyXcodeCore/Commands/AmplifyCommandError.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct AmplifyCommandError: Error {
+    public enum AmplifyCommandErrorType {
+        case unknown
+        case fileNotFound
+        case folderNotFound
+        case xcodeProject
```

Comment:
nit: the other errors pretty clearly say what's going on -- is `xcodeProject` as descriptive as we can get? (I'm assuming this is a type that wraps errors from one of the 3p libraries dealing w/Xcode projects?

Reply:
yes, you're right. I tried to map the errors coming from the 3p libraries with the `XcodeProjectError` enum and the `xcodeProject` case is just a wrapper